vscode/src/jdkDownloader/resources/styles.ts

```diff
@@ -0,0 +1,103 @@
+export const downloaderCss = `
```

Comment:
we can probably follow naming convention like  <jdkDowloader>.styles.css , so that we can reuse same conventions

Reply:
idealy resources should not be part of code, it can be .properties .xml configs etc., see if we can we move this to .css file?

Reply:
There are some limitations of vscode webview due to security purpose de don't allow us adding external references of resources, there is a work around for that, but I didn't wanted to use that that's why kept in .ts format

Reply:
> we can probably follow naming convention like .styles.ts , so that we can reuse same conventions

Ok would rename the file. Thanks